CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.0.2] - TBD
```

Comment:
Let's go with 4.1.0 for the version number since we've changed a lot of dependency versions (please change here and the SNAPSHOT version in the POM)